### DISCUSSION TITLE
Save public battlesnakes

### DISCUSSION THREAD

**Discussion by KlotzAndrew** [BattlesnakeOfficial/feedback#59] [Feature Requests]
Under 'play game' on the `/create` page would it be possible to save battlesnakes?

When playing with friends, every time I hit "play game" I have to find their battlesnake again under the *Find a Public Battlesnake* search tab. Since it is frequently the same battlesnakes that I am playing against, it is the same search I have to do over and over again. It would be a quality of life improvement to click from a list of saved battlesnakes similar to the list of my own snakes, without having to search every time.

**Comment by InterestingBrainPoops** [comment]
Does the rematch button fix your issue? Or do you mean some type of favorite system?

**Reply by KlotzAndrew** [reply]
Some type of favorite system sounds like it. For cases where it's not someone that I'm immediately rematching, potentially battled someone else in the meantime, closed the browser, or some time has passed since I last battled them and I need to remember the name of their snake.

**Comment by bvanvugt** [comment]
Yeah, I think this is a good suggestion. Thanks for making it. Rematch button solves this sometimes, but certainly not in all cases.

We chatted about this internally and were wondering what you thought of the following interim solution:
- When searching/adding public Battlesnakes to your game, maintain a list of Battlesnakes you've added recently
- When creating a new game, display this list along with buttons to "quick add" them to the current game

This would likely be browser/session specific to start, but we could get something live a lot sooner with this approach.

**Reply by KlotzAndrew** [reply]
That sounds like a reasonable first step

**Comment by bvanvugt** [comment]
@KlotzAndrew this has been implemented now, as a list of 'Recently Played Battlesnakes' on the game creation page 👍

**Reply by metawilm** [reply]
Currently on the game creation page there is no such list anymore of 'Recently Played Battlesnakes'. Can we reopen this issue?
